niceday_client/niceday_client.py

```diff
@@ -186,3 +187,47 @@ def get_smoking_tracker(self, user_id: int, start_time: datetime.datetime,
         # convert the json response into a list of dict
         response_json = json.loads(query_response.content)
         return response_json
+
+    def set_tracker_reminder(self, user_id: int, tracker_name: str, reminder_title: str, recurrence_rule: rrule):
+        """
+        Set tracker reminder for a specific user.
+
+        Example Usage:
+            ```
+            client.set_tracker_reminder(12345, TrackerName.SMOKING.value, "This is a tracker", rrule(DAILY,dtstart=datetime.datetime(2022, 5, 12, 0, 0),until=datetime.datetime(2022, 5, 13, 0, 0)))
+
+        Args:
+            user_id: ID of the user we want to set tracker statuses for
+            tracker_name: Name of the tracker to set the reminder for (see
+            https://github.com/senseobservationsystems/goalie-js/issues/840
+            on how to get tracker name, for example: smoking tracker has name=tracker_smoking).
+            Use the TrackerName enum defined in definitions.py instead of hardcoding
+            string values. e.g. 'TrackerName.SMOKING' may be used instead of 'tracker_smoking'.
```

Comment:
```suggestion
            tracker_name: Name of the tracker to set the reminder for (see
                https://github.com/senseobservationsystems/goalie-js/issues/840
                on how to get tracker name, for example: smoking tracker has name=tracker_smoking).
                Use the TrackerName enum defined in definitions.py instead of hardcoding
                string values. e.g. 'TrackerName.SMOKING' may be used instead of 'tracker_smoking'.
```